### PR TITLE
Show null

### PR DESCRIPTION
**Pull request by RFSH**
This PR

- adds support for `show_nulls` display annotations.
- Updates the `_getAnnotationValueByContext` function to support multi level sub-contexts. e.g. lookup `entry/create`.  if not found, lookup `entry`.  if not found, lookup `*`. (This function assumes that application contexts are separated by "/").

Related Issue: #152

### CODE REVIEW

File: js/core.js [hunk 1]
```diff
@@ -1371,6 +1372,43 @@ var ERMrest = (function (module) {
...
+        },
+
+        // find the null value for the column based on context and annotation
+        _getNullValue: function (context) {
+            if (context in this._nullValue) { // use the cached value
+                return this._nullValue[context];
+            }
+
+            var value = -1,
+                displayAnnot = module._annotations.DISPLAY,
```
Comment (howdyjessie) [inline js/core.js]: Why not declare `displayAnnot` and `elements` with `var`s?

Comment (RFSH) [inline js/core.js]: It looks cleaner to me. Avoids having repetitive and unnecessary `var`s.

Comment (howdyjessie) [inline js/core.js]: Gah sorry I knew I was missing something. You're right; this is fine. I just missed the comma at the end of line 1383.

**Comment by jrchudy** [comment]
No specific comments from me. This PR looks ready to merge.

**Pull request merged**